bittide/src/Bittide/ProcessingElement.hs

```diff
@@ -100,9 +106,22 @@ processingElement dumpVcd PeConfig{prefixI, prefixD, initI, initD, iBusTimeout,
     (splitAtCI <| singleMasterInterconnectC) -< (mmDbus, dBus1)
   MM.constBwd prefixD -< dPre
   MM.constBwd prefixI -< iPre
-  wbStorage "DataMemory" initD -< (mmD, dMemBus)
+
+  -- Both the instance and the device types are marked with
+  -- "no-generate" so that a code generator doesn't create wrappers.
+  -- We don't want n different versions of the data and instruction memories
+  -- generated if we don't even want to use them!
```

Comment:
```suggestion
  -- Instruction and data memory are never accessed explicitly by developers, only
  -- implicitly by the CPU itself. We therefore don't need to generate HAL code. We
  -- instruct the generates to skip them by adding a "no-generate" tag.
```

I think the most confusing thing for me was: _why_ don't we want to use them. I guess this is the reason?

Reply:
Yes, that is the reason. We don't need N*2 storage HALs that we never use. It's like accessing `/proc/PID/mem` on Linux, yeah you can access memory that way but you don't usually have a good reason to. I'll update to use your comment! :+1: